backend/app/controllers/books_controller.rb

```diff
@@ -5,7 +5,9 @@ class BooksController < ApplicationController
   before_action :authorize_request
 
   def index
-    render(json: BookSerializer.new(books, meta_pagination(books, { params: { current_user: current_user } })),
+    render(json: BookSerializer.new(books.map do |book|
+                                      book
+                                    end, meta_pagination(books, { params: { current_user: current_user } })),
```

Comment:
`books` returns `Searchkick::Relation` object due to `searchkick`. 

Doing this
```ruby
books.map do |book| book end
```
Satisfies the serializer